src/Roslyn.Suppressions/Roslyn.Suppressions/SonarDiagnosticSuppressor.cs

```diff
@@ -52,6 +53,7 @@ internal SonarDiagnosticSuppressor(Func<IContainer> getContainer)
 
         public override void ReportSuppressions(SuppressionAnalysisContext context)
         {
+            CodeMarkers.Instance.ReportSuppressionsStart();
```

Comment:
This is now the only place we have ETW events that isn't in a try-finally block. However, I'd be inclined to leave it as it is.
I checked, and VS silently catches and squashes any exceptions thrown here (I thought it would report an AD0001 in the Error List, but it doesn't). 
If we see that we are losing "Stop" events we could add a try-finally at that point.